app/src/main/java/com/hedvig/app/feature/embark/ui/EmbarkInsetHandler.kt

```diff
@@ -0,0 +1,42 @@
+package com.hedvig.app.feature.embark.ui
+
+import android.os.Build
+import android.view.View
+import android.view.WindowInsets
+import android.view.WindowInsetsAnimation
+import androidx.annotation.RequiresApi
+import com.hedvig.app.util.ControlFocusInsetsAnimationCallback
+import com.hedvig.app.util.RootViewDeferringInsetsCallback
+import com.hedvig.app.util.TranslateDeferringInsetsAnimationCallback
+
+class EmbarkInsetHandler {
+
+    companion object {
+        @RequiresApi(Build.VERSION_CODES.R)
+        fun setupInsetsForIme(root: View, focusableView: View, vararg translatableViews: View) {
```

Comment:
I'm pretty ok with this being a top-level function or `EmbarkInsetHandler` being an `object` instead of a `class` or perhaps even making this an extension on `View` 😄